titanic.ipynb

```diff
@@ -8,6 +8,7 @@
         "## Titanic 3\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 From line 11 to 15_
```python
plt.title('Survival Rates by Man/Woman/Child')
plt.title('Survival Rates by Man/Woman/Child')
plt.title('Survival Rates by Man/Woman/Child')

# Show see plot
```
From first survival to show see plot

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/10)._

Reply:
a reply